src/pub_verif_token.ts

```diff
@@ -122,6 +127,15 @@ export class TokenResponse implements TokenResponseProtocol {
     }
 }
 
+export function verifyToken(token: Token, publicKeyIssuer: CryptoKey): Promise<boolean> {
```

Comment:
maybe have a static method on `Token` instead. Most of the code is namespaced in classes, but this function

Reply:
I plan to revisit this function once VOPRF tokens lands.